session/bootstrap.go

```diff
@@ -548,6 +548,10 @@ const (
     	key(parent_table_id, create_time),
     	key(create_time)
 	);`
+
+	// CreateTTLResourceGroup creates the resource group for ttl
+	// TODO: set a more proper value for the RU_PER_SECOND
+	CreateTTLResourceGroup = `CREATE RESOURCE GROUP IF NOT EXISTS tidb_ttl RU_PER_SEC=10000000`
```

Comment:
I set a super big value for the `tidb_ttl`, so the ttl is not limited by default. If the users want to limit the TTL, they could alter the resource group `tidb_ttl`. Is it a good pattern?

Reply:
You can user `CREATE RESOURCE GROUP IF NOT EXISTS tidb_ttl RU_PER_SEC=10000 BURSTABLE` instead.